dsp-shared/src/main/scala/dsp/constants/SalsahGui.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package dsp.constants
+
+import dsp.errors._
+
+/**
+ * Contains string constants for IRIs from ontologies used by the application.
+ */
+object SalsahGui {
+
+  /**
+   * `IRI` is a synonym for `String`, used to improve code readability.
+   */
+  type IRI = String
+
+  val InternalOntologyStart = "http://www.knora.org/ontology"
+
+  val SalsahGuiOntologyLabel: String = "salsah-gui"
+  val SalsahGuiOntologyIri: IRI      = InternalOntologyStart + "/" + SalsahGuiOntologyLabel
+  val SalsahGuiPrefixExpansion: IRI  = SalsahGuiOntologyIri + "#"
```

Comment:
Because the two Strings are not IRIs. I add the type though to the one that is missing. 
I copied these constants from the webapi project, so it was like this before.